UnitTests/GitUI.Tests/UserControls/RevisionGrid/Graph/RevisionGraphTests.MoveVisibleAndInvisibleLanesRight_moveFirstLane=False.verified.txt

```diff
@@ -88,14 +88,14 @@
 | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |  /
 | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |   4
 | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |  /
-| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |   5
-| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |  /
-| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |   6
-| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |  /
-| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |   7
-| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |  /
-| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |   8
-| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |  /
-| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |   9
-|/-------------------------------------------------------------------------------------------------´
+| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | 5
```

Comment:
`CreateLane` just increments `LaneCount` and returns the lane number.
Changing this would not save anything.
`MoveLanesRight` is more expensive.

> but I do not see much performance improvements.

Yes, me neither. It is much more important to not call `BuildSegments` at all / i.e. skipping lane straightening if the graph is too wide. For the Linux repo it does not change the time to initial commit if the limit is doubled (`>= 2 * MaxLanes`).